.activate-venv.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# lsh@2022-8-31: per-user settings in settings.yaml
```

Comment:
this should be optional. don't create a file if we don't need one, use sensible defaults in their stead